examples/route_k8s_resource/README.md

```diff
@@ -0,0 +1,126 @@
+# Use any resource as a backing service
```

Comment:
Maybe give it a more specific name like "Binding a nodejs app to a Route/Ingress".